markdown/solace-pubsub-microgateway/solace-pubsub-microgateway.md

```diff
@@ -0,0 +1,1185 @@
+author: Aaron Lee
+summary: Summary of your codelab that is human readable
+id: solace-pubsub-microgateway
+tags: iguide
+categories: Java,REST,API,pub/sub
+environments: Web
+status: Published
+feedback link: Meh! A link where users can go to provide feedback (Maybe the git repo)
```

Comment:
yes add link to github repo with /issues on the end; like this one which was used for battleship: http://github.com/solacetraining/solace-battleship/issues

Reply:
What about Community?  A special forum for Codelabs?

Reply:
So right now at the top of the codelabs page we have this: _Discuss your Codelab experience in the Developer Community_ with the link. I wouldn't create a separate community, but we could create a "Codelabs" category in the current one and direct people directly to that. So the question would be do we create that category now or wait until we see people start to post about the codelabs.

Reply:
As discussed, make a new Category in the Solace Community, and make that the feedback link.  Then we can announce new Codelabs there, etc.